test/cmdlineTests.sh

```diff
@@ -66,11 +72,22 @@ function ask_expectation_update()
 {
     if [ $INTERACTIVE ]
     then
+        if [ $AUTOUPDATE = "true" ]
+        then
+            echo "$newExpectation" > $expectationFile ;
```

Comment:
`newExpectation` and `expectationFile` are referenced before assignment.

Reply:
Looks like the `if [ $AUTOUPDATE = "true" ]` block was inserted in two places by mistake. This should be removed.

Reply:
Fixed.